open-craft/opencraft
# [BB-2299] Fix hero and instance setting text field component

Issue: [BB-2299](https://tasks.opencraft.com/browse/BB-2299)

OCIM UX Fixes
* Instance name and public contact email appearing on Instance settings component
* Hero component empty string text field fixed 

Testing Instructions: 

- For instance, setting component - just visit that page and try to refresh, the instance name and contact email should appear; also try to change the instance name and public contact email and then refresh.
- For hero component fix - just visit the hero component page and empty all title and subtitle to see that it works. 

Reviewers:
